eu.stethewwolf.gresistor.json

```diff
@@ -0,0 +1,48 @@
+{
+    "app-id": "eu.stethewwolf.gresistor",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "gresistor3",
+    "finish-args" : [
+      "--socket=x11"
```

Comment:
you need `--share=ipc` for X11.


also since this appear to be a Gtk3 application, you probably want wayland too.